README.md

```diff
@@ -20,17 +20,11 @@ export default () => {
     url: 'https://example/api'
   });
 
-  if (loading) {
-    return <div>Loading...</div>;
-  }
-
-  if (error) {
-    return <div>{error.message}</div>;
-  }
-
   return {
     <>
-      {data && <div>{data}</div>
+      {loading && <div>'Loading...'</div>}
+      {error && <div>error.message</div>}
+      {data && <div>data</div>}
```

Comment:
Should be `{error.message}`, `{data}`, also `'Loading...'` don't need quotes.

Reply:
Wow, I think the lack of syntax highlighting in the README tripped me up